jormungandr-lib/src/interfaces/value.rs

```diff
@@ -83,6 +83,10 @@ impl<'de> Deserialize<'de> for Value {
     }
 }
 
+#[derive(Deserialize, Serialize)]
+#[serde(transparent, remote = "value::Value")]
+pub struct ValueDef(u64);
```

Comment:
Not going to be always true that a `Value` is a `u64`. Use the `Value` type above and pay the cost of writing `.into()`.

Reply:
This is guaranteed by Serde's `remote` attribute, see https://serde.rs/remote-derive.html

Reply:
I still don't see the point in duplicating that `Value`. I see how it makes it _easy_. for you but I'd rather like to see all of this going toward a harmonised interface.

Reply:
All fields in `SettingsDto` are "normal" types from `chain-impl-mockchain`, but with custom serialization (mostly `with` and `remote` types). This make its usage easier, especially in tests. Adding `remote` type for `Value` lets us keep this property of `SettingsDto`.

I also think that its a cleaner solution than conversion from and to the `jormungandr-lib::Value` and it's generally a good idea to start using it in all Serde DTOs.